core/src/main/resources/jenkins/security/AuthorizationStrategyOverrideConfiguration/config.groovy

```diff
@@ -0,0 +1,36 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2011, CloudBees, Inc.
```

Comment:
This is an almost copy of 
https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/jenkins/security/QueueItemAuthenticatorConfiguration/config.groovy

I'm not sure how to do for the license notice when I copied an existing file.